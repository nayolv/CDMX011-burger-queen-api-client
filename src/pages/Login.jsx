import { Fragment } from "react";
import Banner from "../components/Banner";
import { FormLogin } from "../components/FormLogin";
import camarera from "../images/camarera.png";
import cocinero from "../images/cocinero.png"; 
import gerente from "../images/gerente.png";

function Login() {
  return (
    <Fragment>
      <Banner>
        <p>POS system</p>
      </Banner>
      <main>
        <div className="images">
          <img src={gerente} alt="gerente" />
          <img src={camarera} alt="camarera" />
          <img src={cocinero} alt="cocinero" />
        </div>
        <FormLogin/>
      </main>
    </Fragment>
  );
}
export default Login;
