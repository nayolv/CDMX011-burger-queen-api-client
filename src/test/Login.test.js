import React from 'react';
import renderer from 'react-test-renderer';
import Login from '../pages/Login';


test('DeberÃ­a renderizar el Login', () => {
    const component = renderer.create(<Login/>);
    let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
