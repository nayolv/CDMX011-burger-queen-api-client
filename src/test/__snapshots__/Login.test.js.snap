// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debería renderizar el Login 1`] = `
<main>
  <div
    className="images"
  >
    <img
      alt="Error al cargar imagen"
      src="gerente.png"
    />
    <img
      alt="Error al cargar imagen"
      src="camarera.png"
    />
    <img
      alt="Error al cargar imagen"
      src="cocinero.png"
    />
  </div>
  <form
    onSubmit={[Function]}
  >
    <section
      id="float-father"
    >
      <p>
        Inicia tu sesión
      </p>
      <section
        id="float-label"
      >
        <label
          className=""
          htmlFor="email"
        >
          Correo Electrónico
        </label>
        <input
          id="email"
          onChange={[Function]}
          type="text"
          value=""
        />
      </section>
      <section
        id="float-label"
      >
        <label
          className=""
          htmlFor="email"
        >
          Constraseña
        </label>
        <input
          id="password"
          onChange={[Function]}
          type="password"
          value=""
        />
      </section>
      <button
        id="login"
        type="submit"
      >
        Iniciar sesión
      </button>
    </section>
  </form>
</main>
`;
